pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2709,6 +2739,9 @@ func updatePortAndPciBackIoBundle(ctx *domainContext, ib *types.IoBundle) (chang
 				log.Error(err)
 			}
 		}
+		if ib.Type == types.IoOther && nvmeDeviceIsMountedByEVE(ib.PciLong) {
```

Comment:
I suspect we will introduce a ib.Type for NVME real soon now (we already have it in the protobuf API).
It probably makes sense defining that in this PR and have the code check for both IoOther and IoNVME to avoid forgetting to do that later.

It might make sense to move all of the error logging here as in the above PCIIsBootVga call.

Reply:
How about having IoNVME equal to 255 and changing it later to the correct value?

Reply:
Sure, that works. But we during a transition we might have some models (coming from the controller) using 255 while others use the new value. But we can figure that out later.